overview/glossary.md

```diff
@@ -11,35 +11,82 @@ sidebarType: 0
 - TOC
 {:toc}
 
-## Account ID
+## Ad Formats
 
-`Prebid Server` - Prebid Server host companies may give out account IDs to publishers for authorization, configuration,
-and reporting purposes. This account ID is provided on s2sConfig when utilizing Prebid Server from Prebid.js and in the init function when utilizing Prebid SDK.
+This section contains terms around ad formats and their respective configuration.
 
-## AdSlot
+### MediaType or Media Format
 
-The 'hole' in the web page or mobile app where an ad may appear. This is very similar to `AdUnit`, but is
-the technical implementation defining which ad sizes and formats may appear within it.
+`banner` - also called 'display', these ads are generally simple rectangular images.
 
-In the "Google Publisher Tag" (GPT), the "defineSlot()" function call maps an HTML div (the technical
-implementation) to the AdUnit, (the sellable item).
+`native` - the components of this mediatype like text, links, and images are all separated so they
+can be formatted to smoothly fit into the content.
 
-## AdUnit
+`video` - see Player-Initiated Video and Page-Initiated Video
 
-`Ad servers such as Google Ad Manager` - the AdUnit is the thing that business people care about. It's an
-inventory concept targeted by [line items](#line-items).
+### Multiformat AdUnit
 
-[Prebid Adunit] - Prebid adds another layer to the ad server AdUnit: which header bidders should participate in the auction and their parameters.
+An AdUnit that can display creatives of multiple media formats: banner, native, and/or video.
+
+### Page-Initated Video
+
+Also called 'outstream', these are AdUnits that work in Prebid.js just like banners, but are displayed with
```

Comment:
> AdUnits that work in Prebid.js just like banners

What do you mean by "works just like banners" ? I think this might be misleading, as it sounded to me that outstream video is as easy to setup as banner ads.

Reply:
How about "renders like banners"?